bedrock/firefox/templates/firefox/geolocation.html

```diff
@@ -201,6 +201,20 @@ <h3 data-accordion-role="tab">{{ _('How do I turn off Location-Aware Browsing pe
   </div>
 </section>
 
+<section>
+  {% if l10n_has_tag('geolocation_faq_update') %}
```

Comment:
The l10n tag should be wrapped outside the `<section>` element, else locales where the tag has not been enabled will be left with an empty element.